compiler/modules/front/codeGen.bal

```diff
@@ -760,7 +760,7 @@ function codeGenPanicStmt(CodeGenContext cx, bir:BasicBlock startBlock, Environm
 
 function codeGenVarDeclStmt(CodeGenContext cx, bir:BasicBlock startBlock, Environment env, s:VarDeclStmt stmt) returns CodeGenError|StmtEffect {
     var { varName, initExpr, td, isFinal } = stmt;
-    if lookup(varName, env) !== () {
```

Comment:
Deal with var decl separately

Reply:
Removed the check related to WILDCARD from `codeGenVarDeclStmt` and factored out common part to a separate method to call from `codeGenAssignToWildcard`